recipes/_source.rb

```diff
@@ -36,7 +36,7 @@
 pdns_source_dir = pdns_dir(pdns_filename)
 
 remote_file pdns_filepath do
-  source lazy { node['pdns'][flavor]['source']['url'] }
+  source node['pdns'][flavor]['source']['url']
```

Comment:
This is a pretty old workaround. Are you still using Chef 10 or a version earlier than 11.14?